paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -153,6 +159,9 @@ internal class DefaultEventReporter @Inject internal constructor(
     }
 
     override fun onShowNewPaymentOptions() {
+        CoroutineScope(workContext).launch {
+            analyticEventCallbackProvider.get()?.onEvent(AnalyticEvent.PresentedSheet())
```

Comment:
Do we need to wrap these in a try? What happens if the merchants code throws?

Should we add a fireAnalyticsEvent helper to do this work?